Library/Formula/rtags.rb

```diff
@@ -0,0 +1,38 @@
+class Rtags < Formula
+  desc "ctags-like source code cross-referencer with a clang frontend"
+  homepage "https://github.com/Andersbakken/rtags"
+
+  head "https://github.com/Andersbakken/rtags.git"
+
+  stable do
```

Comment:
Don't need a `stable do` block unless you're doing something that only applies to stable, like a patch or resource.